tests/test_fs.py

```diff
@@ -376,32 +381,51 @@ def test_getinfo(tmppath):
     """Test getinfo."""
     fs = XRootDPyFS(mkurl(tmppath))
 
+    namespaces = [
+        "details",
+        "stat",
+        "lstat",
+        "link",
+        "access",
+        "xrootd"
+    ]
+
     # Info for file
     f = "data/testa.txt"
-    info = fs.getinfo(f)
-    assert info['size'] == os.stat(join(tmppath, f)).st_size
```

Comment:
You're probably already aware so just a note: I think this API is being used in Invenio-XRootD and possible also Invenio-Files-REST

Reply:
Thanks for the heads-up! No, I didn't see it yet, I am still upgrading `invenio-xrootd` (and run tests) to be able to test all these changes in real instance.